CMakeLists.txt

```diff
@@ -180,8 +182,8 @@ find_package(JPEG REQUIRED)
 include_directories(${JPEG_INCLUDE_DIRS})
 
 # Lua
-find_package(Lua)
-if(LUA_FOUND)
+if(WITH_LUA)
```

Comment:
Programmatically you should know what you expect to compile. So, if you want compile libresprite with lua support, you should explicitly define it to buildsystem via -DWITH_LUA=ON, there should no any automagic dependency findings.
`find_package` with version definition finds version equal or greater (unless `EXACT` is defined), so `find_package(Lua 5.2 REQUIRED)` will pass with any version of lua>=5.2.

Reply:
There is no "automagic dependency finding", unless you consider find_package automagic. The idea is that we don't want to have explicit opt-in for what should be a standard feature. We could have explicit opt-out, but that is not very different from what we have now (the feature simply disables itself with a warning instead of throwing an error).